testing/acceptance_test.py

```diff
@@ -915,7 +915,8 @@ def test_calls(self, testdir):
             ["*durations*", "*call*test_3*", "*call*test_2*"]
         )
         result.stdout.fnmatch_lines(
-            ["(0.00 durations hidden.  Use -vv to show these durations.)"]
+            ["(8 items with duration < 0.005s hidden.  Use -vv to show these "
```

Comment:
How come it shows 8 items I wonder? I count 4 items in total, and out of only 2 are supposed to count.

Reply:
That's because it also includes setup/teardown durations, here's the run with `--durations=999 -vv`:

```
===================== slowest 99 test durations =====================
0.02s call     test_durations.py::test_3
0.01s call     test_durations.py::test_2
0.00s call     test_durations.py::test_1
0.00s setup    test_durations.py::test_3
0.00s setup    test_durations.py::test_something
0.00s teardown test_durations.py::test_2
0.00s setup    test_durations.py::test_1
0.00s teardown test_durations.py::test_something
0.00s teardown test_durations.py::test_3
0.00s call     test_durations.py::test_something
0.00s teardown test_durations.py::test_1
0.00s setup    test_durations.py::test_2
========================= 4 passed in 0.08s =========================
```

With `--durations=10` we get:

```
===================== slowest 10 test durations =====================
0.02s call     test_durations.py::test_3
0.01s call     test_durations.py::test_2

(8 items with duration < 0.005s hidden.  Use -vv to show these durations.)
========================= 4 passed in 0.06s =========================
```

Perhaps "items" is a bit misleading? How about "entries" instead?

Reply:
i propose the word `reports` since i believe it may match better

Reply:
@RonnyPfannschmidt , I also prefer "entries" over "reports" : even if the latter is more accurately reflecting the inner objects (`TestReport`), it is a bit misleading for the "lambda" user.

Shall I also modify the `slowest xx test durations` header to remove the word `test` ? 

For example:

```
===================== slowest 10 durations =====================
0.02s call     test_durations.py::test_3
0.01s call     test_durations.py::test_2

(8 entries with duration < 0.005s hidden.  Use -vv to show these durations.)
========================= 4 passed in 0.06s =========================
```

Reply:
@smarie of course, im sure that with more discussion and input we will reach a conclusion that will serve the users best, 

i do have to admit that my bias is structurally technical (aka i like to have the ux and the technicalities to match in such a sense that we do not need to have a dissonance between what appens on technical level and what happens on ux level), but it may be better to change the technical aspect to match the comprehensible ux rather than the other way around

Reply:
I eventually modified it that way:

```
===================== slowest 10 durations =====================
0.02s call     test_durations.py::test_3
0.01s call     test_durations.py::test_2

(8 durations < 0.005s hidden.  Use -vv to show these durations.)
========================= 4 passed in 0.06s =========================
```

Let me know if that works for everyone

Reply:
I like it :+1: